src/geo/src/GeoPMTFactoryBase.cc

```diff
@@ -253,19 +280,19 @@ namespace RAT {
 
 	// Add Light cone geometry from Sheffield
         G4Paraboloid* lightcone_outer = new G4Paraboloid("lightcone_outer",
-                                                        17.163*CLHEP::cm,
-                                                        0.5*25.5*CLHEP::cm,
-                                                        0.5*42.081*CLHEP::cm);
+                                                        light_cone_length*CLHEP::cm,
+                                                        light_cone_innerradius*CLHEP::cm,
+                                                        light_cone_outerradius*CLHEP::cm);
         G4Paraboloid* lightcone_inner = new G4Paraboloid("lightcone_inner",
-                                                        17.363*CLHEP::cm,
-                                                        0.5*25.3*CLHEP::cm,
-                                                        0.5*41.881*CLHEP::cm);
+                                                        (light_cone_length+2)*CLHEP::cm,
+                                                        (light_cone_innerradius-2)*CLHEP::cm,
+                                                        (light_cone_outerradius-2)*CLHEP::cm);
```

Comment:
Just a few questions: 
1. Since you've already managed to put in ratdb entries for all other parameters, might as well remove the magic "2", and replace it with something like light_cone_thickness.
2. Does lightcone_inner actually have to be longer than lightcone_outer?

Reply:
1. i can add the light cone thickness variable, good idea
2. yes, its a property of the paraboloid function apparently, that's why i made inner cone longer than outer cone